### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mcardoso3-stat6250**
My week 1 quiz answers

### CODE REVIEW

File: week-01/README.md [hunk 1]
```diff
@@ -14,44 +14,43 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Structure Quiz, Problem 1]
-
-
+- Question (mcardoso3ˆ’stat6250): Is one component of the Week 1 Setup instructions more important to comprehend than the others for the purpose of understanding the Initial Setup Achievement?
```
Comment (prof-lankham) [inline week-01/README.md:17]: Your username should appear in the parentheses for each question/answer, exactly as it appears in GitHub.

File: week-01/README.md [hunk 2]
```diff
@@ -14,44 +14,43 @@ The instructor will then review the pull request and make comments should furthe
...
-
-
+- Question (mcardoso3ˆ’stat6250): What are the requirements to earning "complete" mark in the Weekly Forum Posts?
+- Answer (mcardoso3ˆ’stat6250):  Earning a "complete" mark in the Weekly Forum posts require including the title, complete URL, synopsis, and article choice along with the proper amount of detail provided.
 
 [Course Structure Quiz, Problem 3]
-
-
+- Question (mcardoso3ˆ’stat6250):  Is the submitting the weekly quizzes portion a more crucial concept to look over more than the formulation of the quizzes and answers?
+- Answer (mcardoso3ˆ’stat6250):
```
Comment (prof-lankham) [inline week-01/README.md:25]: Empty answer blocks should be deleted.

**Comment by prof-lankham** [comment]
Please address comments and resolve conflicts before this PR can be accepted.

**Comment by prof-lankham** [comment]
To keep this PR from becoming stale, please address feedback, resolve merge conflicts (by rebasing your fork), and submitting a new PR.

**Comment by mcardoso3-stat6250** [comment]
Thank you for informing me of these changes.  I  will try to resubmit as soon as possible.

**Pull request closed without merging**